dataplane/linux/ipip_mgr.go

```diff
@@ -224,9 +224,7 @@ func (m *ipipManager) CompleteDeferredWork() error {
 		for _, ip := range m.activeHostnameToIP {
 			members = append(members, ip)
 		}
-		for _, ip := range m.externalNodeCIDRs {
-			members = append(members, ip)
-		}
+		members = append(members, m.externalNodeCIDRs...)
```

Comment:
Same change just above?

Reply:
Fixed

Reply:
Actually, `m.activeHostnameToIP` is a map.. I'm not sure how to use it in an `append()` call (and neither does any of the folks around here I could find atm)